src/runtime_src/core/common/drv/include/kds_client.h

```diff
@@ -87,8 +103,14 @@ struct kds_client {
 	struct mutex		  lock;
 
 	/* TODO: xocl not suppot multiple xclbin context yet. */
-	struct kds_client_ctx    *ctx;
-	struct list_head          ctx_list;
+	struct kds_client_ctx    	*ctx;
+
+	/* To suppot ZOCL  multiple PL support */
+	struct list_head          	ctx_list;
+
+	/* To suppot multiple hw context */
+	struct list_head          	hw_ctx_list;
+	uint32_t 		 	next_avail_hw_ctx;
```

Comment:
In the use case, it looks like this is use for hw_ctx id. To make the name clear, how about name it as hw_ctx_id_counter or next_hw_ctx_id?

Reply:
Thanks for the suggestion. Updated accordingly.